File: perl/v1_soap.pl
#!/usr/bin/perl
# v1_soap.pl
# Instalar: cpan SOAP::Lite
# Ejecutar: perl v1_soap.pl 10

use strict;
use warnings;
use SOAP::Lite;

my $wsdl = 'https://www.dataaccess.com/webservicesserver/NumberConversion.wso?WSDL';
my $num  = $ARGV[0] // 10;

my $client   = SOAP::Lite->service($wsdl);
my $response = $client->NumberToWords($num);
print "$response\n";

File: nodejs/v1_soap.js
// v1_soap.js
// Instalar: npm install
// Ejecutar: node v1_soap.js 10

const soap = require('soap');

const WSDL = 'https://www.dataaccess.com/webservicesserver/NumberConversion.wso?WSDL';
const num  = parseInt(process.argv[2]) || 10;

soap.createClient(WSDL, (err, client) => {
    if (err) throw err;
    client.NumberToWords({ ubiNum: num }, (err, result) => {
        if (err) throw err;
        console.log(result.NumberToWordsResult);
    });
});
